client/.storybook/stories/Notifications.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { useNotification } from 'hooks/useNotification'
+import { Box } from 'grommet'
+import { Button } from 'components/Button'
+import { Notifications } from 'components/Notifications'
+
+export default {
+    title: 'Components/Notifications'
+}
+
+export const Default = () => <Notifications />
```

Comment:
```suggestion
export const Default = () => {
  const { showNotification } = useNotification()
  return (
    <Notifications />
    <Button onClick={() => showNotification(id, type, message)}/>
  )
}
```

if you want to switch the order, i would make it message, id, type where id is auto set as `Date.now()` and type should be `success` in the function definition